qemu/tests/cfg/win_virtio_driver_installer_pretest.cfg

```diff
@@ -0,0 +1,125 @@
+        nic_model_nic1 = e1000e
+    # install winfsp tool
+    i386, i686:
+        install_winfsp_path = 'C:\Program Files'
+    x86_64:
+        install_winfsp_path = 'C:\Program Files (x86)'
+    nics += " nic2"
+    nic_model_nic2 = virtio
+    no_virtio_rng:
+        virtio_rngs += " rng0"
+        backend_rng0 = rng-builtin
+        backend_type = builtin
+    images += " stg0 stg1"
+    image_name_stg0 = "images/stg0"
+    image_name_stg1 = "images/stg1"
+    image_size_stg0 = 5G
+    image_size_stg1 = 10G
+    drive_format_stg0 = virtio
+    drive_format_stg1 = scsi-hd
+    remove_image_stg0 = yes
+    remove_image_stg1 = yes
+    force_create_image_stg0 = yes
+    force_create_image_stg1 = yes
+    serials += " vs"
+    serial_type_vs = virtserialport
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    install_balloon_service = "%s:\Balloon\blnsvr.exe -i"
+    uninstall_balloon_service = "%s:\Balloon\blnsvr.exe -u"
+    status_balloon_service = "%s:\Balloon\blnsvr.exe status"
+    run_balloon_service = "%s:\Balloon\blnsvr.exe -r"
+    stop_balloon_service = "%s:\Balloon\blnsvr.exe -s"
+    inputs = input1
+    input_dev_bus_type_input1 = virtio
+    input_dev_type_input1 = mouse
+    vmcoreinfo = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir = virtio_fs_test/
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
+    mem_devs = mem1
+    backend_mem_mem1 = memory-backend-file
+    mem-path_mem1 = /dev/shm
+    size_mem1 = 4G
+    use_mem_mem1 = no
+    share_mem = yes
+    guest_numa_nodes = shm0
+    numa_memdev_shm0 = mem-mem1
+    numa_nodeid_shm0 = 0
+    fs_binary_extra_options = " -o cache=auto"
+    test_drivers = 'viorng viostor vioscsi viofs balloon'
+    driver_test_name_viorng = 'rng'
+    driver_test_name_viostor = 'iozone'
+    driver_test_params_viostor = {'images': 'stg0'}
+    driver_test_name_vioscsi = 'iozone'
+    driver_test_params_vioscsi = {'images': 'stg1'}
+    driver_test_name_viofs = "viofs_basic_io"
+    driver_test_name_balloon = "balloon"
+    iozone_cmd_opitons = " -azR -r 64k -n 512M -g 1G -M -I -i 0 -i 1 -b iozone.xls -f %s:\testfile"
+    read_rng_cmd = "WIN_UTILS:\\random_%PROCESSOR_ARCHITECTURE%.exe"
+    driver_test_names = 'balloon viofs_basic_io'
+    exe_file_name = "virtiofs.exe"
+    exe_find_cmd = 'dir /b /s VIOWIN_LTR\\EXE_FILE_NAME'
+    inf_find_cmd = 'dir /b /s VIOWIN_LTR\\DRIVER_NAME.inf'
+    variants:
+        - create_iso:
+            create_iso = yes
+            driver_folder_name = Virtio-Win
+            guest_path = "C:\Program Files\${driver_folder_name}\*"
+            host_installer_file = "/home/installer"
+            host_iso_path = "/home/virtio-win-iso/"
+            mkiso_cmd = "mkisofs -o /home/virtio-win-pretest.iso -input-charset iso8859-1 -J -R -V "Virtio-Win" ${host_iso_path}"
```

Comment:
Would you please share where do you finally use `/home/virtio-win-pretest.iso`? If it is referenced somewhere else, please make it variable.

Reply:
Yes, it's cdrom_virtio.